packages/flutter_tools/lib/src/ios/simulators.dart

```diff
@@ -610,12 +606,8 @@ class IOSSimulator extends Device {
     }
   }
 
-  bool get _xcodeVersionSupportsScreenshot {
-    return _xcode.majorVersion > 8 || (_xcode.majorVersion == 8 && _xcode.minorVersion >= 2);
```

Comment:
I just noticed we don't need this anymore, the minimum version has been > 8 for awhile.